### DISCUSSION TITLE
Copier does not create .copier-answers.yml in Azure repos

### DISCUSSION THREAD

**Discussion by sandeeps311** [copier-org/copier#1054] [Q&A] [answered]
can anyone please let me know why copier does not create .copier-answers.yml when I'm trying to clone repo using copier ?

**Comment by sandeeps311** [comment, marked as answer]
I haven't added {{_copier_conf.answers_file}} previously in template repo
problem is solved now

**Comment by sisp** [comment]
Related to #983.
